docs/en/docs/tutorial/handling-errors.md

```diff
@@ -163,7 +163,7 @@ path -> item_id
 !!! warning
```

Comment:
@tamird due to the lack of subclassing, the text version of the validation above this is also not the expected output. See: https://github.com/fastapi/fastapi/discussions/12125

Probably makes sense to update this text version as well.

Reply:
Sorry, can you suggest a patch? It has been months since I submitted this PR.

Reply:
Updating the text is not the right solution. See https://github.com/fastapi/fastapi/pull/12972.